### DISCUSSION TITLE
how to proxy 'http://domain.com/api/setScore' to 'http://localhost:3001/setScore'

### DISCUSSION THREAD

**Discussion by KingMatrix1989** [chimurai/http-proxy-middleware#550] [Q&A]
Hi. I want to make a proxy to following rediraction:
**From**:
'http://domain.com/api/setScore'

**To**:
'http://localhost:3001/setScore'

**Comment by AlvinYuXT** [comment]
Do you mean proxy 'http://localhost:3001/setScore' -> 'http://domain.com/api/setScore'
you can do like this:
{
  context: '/setScore',
  target: 'http://domain.com',
  pathRewrite: ['/', '/api']
}

**Comment by sroettering** [comment]
I think he wants it to proxy it the other way around.
I have the same question right now.

**Comment by dobesv** [comment]
You would have to have "domain.com" point to a server you are controlling at that moment (localhost, perhaps, or an alternate local IP like 127.0.0.2, or a docker/vagrant host).  Then you run the proxy on port 80 on that host that sends traffic to localhost port 3001/api.
